subworkflows/nf-core/vcf_joint_calling_germline_gatk/meta.yml

```diff
@@ -0,0 +1,48 @@
+name: "vcf_joint_calling_germline_gatk"
+## TODO nf-core: Add a description of the subworkflow and list keywords
+description: Sort SAM/BAM/CRAM file
+keywords:
+  - sort
+  - bam
+  - sam
+  - cram
+## TODO nf-core: Add a list of the modules used in the subworkflow
+modules:
+  - samtools/sort
+  - samtools/index
+## TODO nf-core: List all of the variables used as input, including their types and descriptions
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - bam:
+      type: file
+      description: BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+## TODO nf-core: List all of the variables used as output, including their types and descriptions
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - bam:
+      type: file
+      description: Sorted BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+  - bai:
+      type: file
+      description: BAM/CRAM/SAM samtools index
+      pattern: "*.{bai,crai,sai}"
+  - csi:
+      type: file
+      description: CSI samtools index
+      pattern: "*.csi"
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+authors:
+  - "None"
```

Comment:
I think you can give credits to both @nickhsmith and yourself for that

Reply:
Will do! This is still very much a WIP though. I'm waiting for the bed scatter subwf

Reply:
```suggestion
  - "nickhsmith"
  - "@matthdsm"
```